chainer_chemistry/saliency/visualizer/mol_visualizer.py

```diff
@@ -46,9 +46,24 @@ def __init__(self, logger=None):
 
     def visualize(self, saliency, mol, save_filepath=None,
                   visualize_ratio=1.0, color_fn=red_blue_cmap,
-                  scaler=abs_max_scaler, legend=''):
+                  scaler=abs_max_scaler, legend='', raise_import_error=True
```

Comment:
I think `raise_import_error` should be `False` by default if we follow the original behavior of saving png files.

Reply:
okay i will fix it to set False as default.